ViewModel/AggregatedStageDamage/ManualStageDamageVM.cs

```diff
@@ -192,9 +192,28 @@ public List<StageDamageCurve> GetStageDamageCurves()
 
         public bool ValidateForm()
         {
+            //are there at least 2 points per curve
+            bool atLeast2Points = AreThereTwoPointsPerCurve();
             bool areRowsUnique = AreManualRowsUniqueCombinations();
             bool curvesValid = AreManualCurvesValid();
-            return areRowsUnique && curvesValid;
+            return areRowsUnique && curvesValid && atLeast2Points;
+        }
+
+        private bool AreThereTwoPointsPerCurve()
+        {
+            bool atLeastTwoPoints = true;
+            foreach (ManualStageDamageRowItem r in Rows)
```

Comment:
I would do this differently, iterate over all the rows and keep track of the unique names of each one that fails the check.  then at the end when you show the message box, you list out the ones that were failing.  This will give the user a better understanding of what's failing and why.

Reply:
Something to the effect of:
The following manually entered curves must have at least 2 points:
- Curve A
- Curve B
- ...

Reply:
Something like:
string.Join(Environment.NewLine + "\t* ", list)

where list is your list of curves.

Reply:
I followed your advice and it now looks like this:
![image](https://user-images.githubusercontent.com/47757848/147957335-215e852c-9de2-45d7-988f-d82ac65f0686.png)

Reply:
Could we go with ALT+0183 or \u2022 as the bullet point instead of *?  I find that it's cleaner looking than *.  If you don't think that's necessary that's fine.  It looks good as is but I think this adds polish.

ALT+0183 is a bit different than \u2022, \u2022 is an escape sequence so you just do "\t\u2022" instead of "\t*".  ALT+0183 needs to be typed by pressing and holding ALT, then typing the number, and letting go of ALT, and it'll add the character directly to the string instead of an escape sequence.  If you use \u2022, make a comment about how it's a bullet point character, because it helps clarify what the character is intended to be.

Reply:
I added the \u2022.
![image](https://user-images.githubusercontent.com/47757848/148402186-c11e6916-7af6-4e59-913c-254b38528f18.png)